app/templates/search/macros/filters/date_input.html

```diff
@@ -0,0 +1,37 @@
+{% from 'components/button/macro.html' import tnaButton %}
+{% from 'components/date-search/macro.html' import tnaDateSearch %}
+{% from "components/date-input/macro.html" import tnaDateInput %}
+
+
+{% macro date_input(request, name, id, title) %}
+<div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
+  <h3 class="tna-heading-m">{{ title }}</h3>
+    {{ tnaDateInput({
+    'label': "Enter a start date",
+    'headingLevel': 4,
+    'headingSize': "m",
+    'id': "date",
+    'name': "date"
+    }) }}
```

Comment:
Do these need `'progressive': true`?

Do they also need to be populated with the current posted date?